app/assets/stylesheets/custom.css.scss

```diff
@@ -152,3 +152,36 @@ p {
     color: #333;
   }
 }
+
+/* forms */
+
+input, textarea, select, .uneditable-input {
+  border: 1px solid #bbb;
+  width: 100%;
+  margin-bottom: 15px;
+  @include box_sizing;
+}
+
+input {
+  height: auto !important;
+}
+
+label.required:after {
+  color: $purple;
+  content:" *";
+}
+
+#error_explanation {
+  color: red;
```

Comment:
Color `red` should be written in hexadecimal form as `#ff0000`<br>Rule sets should be ordered as follows: @extends, @includes without @content, properties, @includes with @content, nested rule sets<br>Properties should be ordered background, border, color, margin-bottom